src/NetscriptFunctions.ts

```diff
@@ -1913,6 +1909,34 @@ export const ns = {
   ...NetscriptExtra(),
 };
 
+const wrappedNS = wrapAPILayer({} as ExternalAPI<NSFull>, ns, []);
+function freezeEnums(obj: object, isEnums = false) {
```

Comment:
Why not do this in wrapAPILayer, which is already doing this traversal *and* checking for enums?

Reply:
Moved enum freezing to wrapAPILayer